mibs/trango/TRANGO-OrionLynx-MIB-COMPACT.mib

```diff
@@ -0,0 +1,618 @@
+GIGA-PLUS-MIB DEFINITIONS ::= BEGIN

```

Comment:
You can rename this mib file to be `GIGA-PLUS-MIB DEFINITIONS` so it's obvious what it is.